internal/provider/resource_storagezone.go

```diff
@@ -26,6 +26,8 @@ const (
 	keyRewrite404To200    = "rewrite_404_to_200"
 )
 
+	var RegionException		= []string{"SYD", "SG", "BR"}
```

Comment:
Instead of a string slice, this should be a map[string]struct{}{}.
It simplifies the code when doing the check (`stringInSlice` won't be needed) and it is more performant to check if a region is in the slice/map.